buildSrc/src/main/kotlin/configure-compilation-conventions.gradle.kts

```diff
@@ -1,9 +1,19 @@
 import org.jetbrains.kotlin.gradle.tasks.*
+import kotlin.reflect.*
+import kotlin.reflect.full.*
 
 configure(subprojects) {
     val project = this
     if (name in sourceless) return@configure
     apply(plugin = "org.jetbrains.kotlinx.atomicfu")
+    // Workaround for KT-71203. Can be removed after https://github.com/Kotlin/kotlinx-atomicfu/issues/431
+    afterEvaluate {
```

Comment:
The problem is that `AtomicFUPluginExtnesion` type is not accessible in the root build script yet, and seems that the only way to dynamically set the property in this extension class is via reflection 🥲

Reply:
How about something like this? https://github.com/Kotlin/kotlinx.coroutines/commit/60c070771a7fdcd2b364fcc19f4b272f2ce9211f

Reply:
Cool! Thanks, that's much better and solves the problem)